spark/dl/src/main/scala/org/apache/spark/ml/DLEstimator.scala

```diff
@@ -271,7 +271,7 @@ object DLModel {
 trait HasBatchSize extends Params {
 
   final val batchSize: Param[Int] = new Param[Int](this, "batchSize", "batchSize")
-  setDefault(batchSize -> 1)
+  setDefault(batchSize -> 4)
```

Comment:
different default value in python and scala?

Reply:
changed back to the same